mvp/resources/README

```diff
@@ -1 +1,71 @@
-Dummy readme
\ No newline at end of file
+IBM Distribution for Galasa version 1.0 (20210618)                                                                                                                
+===============================================================================      
+
+Overview                    
+------------                                                                                                                                     
+                                                                                                              
+Start using Galasa to enable deep integration testing across platforms and technologies, and run 
+repeatable, reliable, agile testing at scale across your enterprise.   
+
```

Comment:
We should include that this zip is a network free install of Galasa removing the requirement to connect to the internet when building or running tests etc

Reply:
done